WoodenUtilities/src/main/java/com/bedrocklegends/woodenutilities/setup/WoodenBlocks.java

```diff
@@ -16,11 +18,14 @@
 
 public class WoodenBlocks {
     public static final DeferredRegister<Block> BLOCKS = DeferredRegister
-        .create(ForgeRegistries.BLOCKS, WoodenUtilities.ID);
+            .create(ForgeRegistries.BLOCKS, WoodenUtilities.ID);
     public static final RegistryObject<BasicBlock> WOODEN_TANK = BLOCKS
-        .register("wooden_tank", () -> new WoodenTankBlock(BlockBuilder.create(Material.WOOD).sound(SoundType.WOOD)
-            .hardnessAndResistance(2.0F, 3.0F).harvestTool(ToolType.AXE).harvestLevel(0)
-            .tileEntitySupplier(WoodenTankTile::new)));
+            .register("wooden_tank", () -> new WoodenTankBlock(BlockBuilder.create(Material.WOOD).sound(SoundType.WOOD)
+                    .hardnessAndResistance(2.0F, 3.0F).harvestTool(ToolType.AXE).harvestLevel(0)
+                    .tileEntitySupplier(WoodenTankTile::new)));
+    public static final RegistryObject<FlowingFluidBlock> RESIN = BLOCKS.register("resin", () ->
```

Comment:
Can we move the `"resin"` string literal to the `WoodenConstants` class under an inner `Blocks` class so that we can reference it everywhere with a variable rather than a literal?

Reply:
Done in 803f9eb